fixql-core/src/test/scala/com/iterable/graphql/BuilderDslSpec.scala

```diff
@@ -35,6 +35,7 @@ class BuilderDslSpec extends FlatSpec with SchemaAndMappingsMutableBuilderDsl wi
       field("humans", list(humanType)) ~> QueryReducer.jsObjects { _ =>
         DBIO.successful(repo.getHumans(1000, 0).map(Json.toJson(_).as[JsObject]))
       }
+          .toTopLevelArray
```

Comment:
The changes in this file are an unrelated fix to a test. Top-level fields that return lists of objects (which is very common) need to use this helper method.